.github/workflows/docker.yaml

```diff
@@ -0,0 +1,48 @@
+name: Docker
+on:
+  push:
+    branches:
+      - main
+
+env:
+  VERSION: 4.8.12
```

Comment:
Should we use the GitHub SHA as part of the tag instead? Or can we extract the version from one of the files that `bumpversion` updates?

Reply:
`bump2version` updates this value directly